Core/MemMap.cpp

```diff
@@ -194,7 +194,11 @@ static bool Memory_TryBase(u32 flags) {
 			*view.out_ptr = *views[i - 1].out_ptr;
 		} else {
 			*view.out_ptr = (u8*)g_arena.CreateView(
+#ifdef _ARCH_32
 				position, view.size, base + (view.virtual_address & MEMVIEW32_MASK));
+#else
+				position, view.size, base + view.virtual_address);
+#endif
```

Comment:
Instead, you probably want to create the view as above in the _M_X64/ARM64 case for 64 bit platforms.

You won't want to ignore views there.  If you are ignoring views, you want the MEMVIEW32_MASK.

-[Unknown]